pkg/gui/controllers/jump_to_side_window_controller.go

```diff
@@ -10,15 +10,18 @@ import (
 
 type JumpToSideWindowController struct {
 	baseController
-	c *ControllerCommon
+	c           *ControllerCommon
+	nextTabFunc func() error
```

Comment:
I dislike indirections through function pointers if we can avoid them, as it makes the code harder to read. I would find it cleaner to extract the `handleNextTab` function to some helper so that we can call it from both places. However, I found this a bit tricky because it depends on some private `Gui` stuff, and I didn't want to invest more time in this.